src/Symfony/Bridge/PhpUnit/ClockMock.php

```diff
@@ -108,10 +96,8 @@ public static function gmdate($format, $timestamp = null)
         return \gmdate($format, $timestamp);
     }
 
-    /**
-     * @return array|int|float
-     */
-    public static function hrtime($asNumber = false)
+
+    public static function hrtime($asNumber = false): array|int|float
```

Comment:
The min PHP version of the phpunit-bridge is PHP 7.2 (this bridge has a special status regarding the min PHP version as we use it to test older maintained branches). So the union type cannot become a native return type.